AGILE/AnimatedObject.cs

```diff
@@ -938,7 +938,7 @@ private bool Collide()
         /// <returns></returns>
         private byte CalculatePriority(int y)
         {
-            return (byte)(y < state.PriorityBase ? Defines.BACK_MOST_PRIORITY : (byte)(((y - state.PriorityBase) / ((168.0 - state.PriorityBase) / 10.0f)) + 5));
+            return (byte)(y < state.PriorityBase ? 4 : (byte)(((y - state.PriorityBase) / ((168.0 - state.PriorityBase) / 10.0f)) + 5));
```

Comment:
This looks like it might be a merge conflict issue, as it is reverting use of the constant that I added. I don't think you would have modified the AnimatedObject class, so I'll change these lines back to how they were.